package.json

```diff
@@ -6,6 +6,7 @@
     "scripts": {
         "build": "NODE_ENV=production ./node_modules/.bin/webpack --progress --color",
         "build-watch": "npm run build -- --watch",
+        "declaration:build": "rm -f dist/**/*.d.ts lib/**/d.ts && npx tsc -p tsconfig.lib.json; npx tsc -p tsconfig.index.json; mv dist/index.d.ts dist/reactist.d.ts",
```

Comment:
Let's stick with the current pattern of `build-x` for these npm script names

Reply:
Also, can we add a ["types" field](https://www.typescriptlang.org/v2/docs/handbook/declaration-files/dts-from-js.html#editing-the-packagejson) for completeness? TS is able to find reactist.d.ts but it always helps to be explicit 👍

Reply:
> Let's stick with the current pattern of `build-x` for these npm script names

Renamed to `build-types`. Please use github "suggestion" for these simple fixes.

Reply:
> Also, can we add a ["types" field](https://www.typescriptlang.org/v2/docs/handbook/declaration-files/dts-from-js.html#editing-the-packagejson) for completeness? TS is able to find reactist.d.ts but it always helps to be explicit 👍

TBH I don't understand why we have two different kinds of output and how this relates. it we include it won't it mess up `lib` usage?

Reply:
We don't use the "libs" dist build in Twist or Todoist, but you can also consume the library like `import { Avatar } from "@doist/reactist", and that will tell webpack/typescript to point to the "main" and "types" fields in package.json. The difference is the entire library will end up being bundled into the app's output.

Reply:
According to documentation it will use `main` field, no need to duplicate it.